helm-swoop.el

```diff
@@ -534,8 +534,7 @@ If $linum is number, lines are separated by $linum"
          (mapconcat 'identity
                     (split-string helm-pattern " ")
                     "\\|")))
-    (when (or (and (featurep 'migemo) helm-migemo-mode
-                   (migemo-forward $regex nil t))
```

Comment:
One line is perfectly fine, plz rebase it on original commit.

Reply:
OK. I will.